### PR TITLE
Unit tests

### PR DESCRIPTION
**Pull request by Xyarz**

### CODE REVIEW

File: R/posterior.R
Comment (sebastianbossert) [inline R/posterior.R:1]: Please revert the changes around ifels for mu_hat, se_hat and data. Either mu_hat &se_hat or data should be specified as input.

Comment (Xyarz) [reply R/posterior.R:1]: changed

**Review by sebastianbossert — APPROVED** [review]
